### ISSUE TITLE
Newly generated basic-ts app doesn't compile.

### ISSUE THREAD

**Issue by josh-barker** [probot/create-probot-app#218]
Hi,

Maybe I missed something in the documentation, but I can't seem to get a basic-ts example to compile.
VS Code also shows that it cannot find `context.issue` and `context.github`.

> Property 'issue' does not exist on type 'WebhookEvent<WebhookPayloadIssues>'.ts(2339)
> Property 'github' does not exist on type 'WebhookEvent<WebhookPayloadIssues>'.ts(2339)

What else do I need to do to get it to work?

Cheers

### Details:

```shell
npm --version
6.14.7

node --version
v12.16.1

npx tsc
src/index.ts(5,34): error TS2339: Property 'issue' does not exist on type 'WebhookEvent<WebhookPayloadIssues>'.
src/index.ts(6,19): error TS2339: Property 'github' does not exist on type 'WebhookEvent<WebhookPayloadIssues>'.
```

index.ts
```typescript
import { Application } from 'probot' // eslint-disable-line no-unused-vars

export = (app: Application) => {
  app.on('issues.opened', async (context) => {
    const issueComment = context.issue({ body: 'Thanks for opening this issue!' })
    await context.github.issues.createComment(issueComment)
  })
  // For more information on building apps:
  // https://probot.github.io/docs/

  // To get your app running against GitHub, see:
  // https://probot.github.io/docs/development/
}
```

**Comment by gr2m** [comment]
Sorry for that, the problem should be resolved now via https://github.com/probot/probot/pull/1374 and [probot v10.9.2](https://github.com/probot/probot/releases/tag/v10.9.2), can you try again please?

**Comment by gr2m** [comment]
There was another problem, it got fixed in 10.9.3.

I've also bumped the minimal required probot version in https://github.com/probot/create-probot-app/pull/216

**Comment by josh-barker** [comment]
Hey @gr2m, yep that fixed the issues.

Thanks!
